aptos-move/aptos-gas/src/aptos_framework.rs

```diff
@@ -184,5 +184,9 @@ crate::natives::define_gas_parameters_for_natives!(GasParameters, "aptos_framewo
     [.object.exists_at.base, { 7.. => "object.exists_at.base" }, 250 * MUL],
     // These are dummy value, they copied from storage gas in aptos-core/aptos-vm/src/aptos_vm_impl.rs
     [.object.exists_at.per_byte_loaded, { 7.. => "object.exists_at.per_byte_loaded" }, 1000],
-    [.object.exists_at.per_item_loaded, { 7.. => "object.exists_at.per_item_loaded" }, 8000]
+    [.object.exists_at.per_item_loaded, { 7.. => "object.exists_at.per_item_loaded" }, 8000],
+    [.debug.native_print.base_cost, {8.. => "debug.native_print.base_cost"}, MUL],
```

Comment:
Did you mean to leave this as MUL?

Reply:
Maybe add 1 * MUL

Reply:
Let's just change this to 0 to avoid confusion. 0 is fine since the functions are no-ops in production.

Reply:
Removed it totally